guides/doc-Content_Management_Guide/topics/Managing_Activation_Keys.adoc

```diff
@@ -72,7 +72,7 @@ To create an activation key, complete the following steps:
 . If you want to set a limit, clear the *Unlimited hosts* check box, and in the *Limit* field, enter the maximum number of systems you can register with the activation key. If you want unlimited hosts to register with the activation key, ensure the *Unlimited Hosts* check box is selected.
 . In the *Description* field, enter a description for the activation key.
 . From the *Environment* list, select the environment to use.
-. From the *Content View* list, select a Content View to use. If you want to use this activation key to register hosts, the Content View must contain the {Project} Tools repository because it is required to install the `katello-agent`.
+. From the *Content View* list, select a Content View to use. If you want to use this activation key to register hosts, the Content View must contain the `{RepoRHEL7ServerSatelliteToolsProductVersion}` repository because it is required to install the `katello-agent`.
```

Comment:
Hi @mcorr, does this solution look elegant to you? The attribute for the satellite-tools repo exists, so I just reused this attribute in all three builds (foreman, foreman-deb, satellite).